apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper_connector.erl

```diff
@@ -273,6 +273,8 @@ do_query(
     Result =
         case try_render_message(Query, Channels) of
             {ok, Msg} ->
+                [{ChannelID, _} | _] = Query,
```

Comment:
always the same channel id for all items in the batch ?

Reply:
Yes